### PR TITLE
ZH-253

### PR DESCRIPTION
**Pull request by silviabah**
## Summary

Adding the ISM documentation and schemas
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New content (non-breaking change which adds new content)
- [ ] Modified content (non-breaking change which modifies existing content)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Motivation and Context
Adding the ISM documentation and schemas

### References
[ZenHub #253](https://app.zenhub.com/workspaces/t43-gdi-sops-662a0b0fe38b0405fb600d67/issues/zh/253)

## Review
Documentation that will be modified in the future as some aspects of the SOP life cycle are define. 

## Checklist:

**General Compliance:**
- [x] My changes follow the code style of this project ([GDI SOP Style Guide](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_style-guide.md)) and the file naming conventions of the [file accessioning proposal](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs//GDI-SOP_sop-accessioning.md).
- [x] I have verified that all new updated content is accessible, including checking that all external references are readable (i.e., no broken links). These may include references to external resources that should be resolvable, and internal references among SOPs.
- [x] I have properly added this PR's changes to the repository [CHANGELOG.md](../CHANGELOG.md).

Only applicable if the PR includes new, or changes to, GDI SOPs (i.e., documents at ``sops/``):
- [ ] My SOP-related changes adhere to the [Generic SOP Template](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_sop-template.md), including format and required fields.
- [ ] I have consulted the [Charter](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_charter.md), [ISM](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_information-service-management.md), and [ORR](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_organisational-roles-and-responsibilities.md) documents to ensure compliance. 
- [x] I am complying with the established procedure for SOP creations and modifications, including respecting review phases and notifying needed contributors for reviews.

### CODE REVIEW

File: docs/GDI-SOP_ISM.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+2. OC/SDPC rep (or nominated experts) uses the template to create the SOP according to their node’s needs and internally review for completeness
+3. OC/SDPC rep initiates their node-defined approval process for the node-specific SOP
+4. [Node-appointed approver] reviews the SOP and approves or requests changes (repeat from Step 2)
+5. OC/SDPC members from the node accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP according to agreed process and the SOP goes into production
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Instance.png)
+
+_Figure 3: Proposal for creating and approving node-specific SOPs instances._
+
+SOP template reviews and revisions should follow guidelines for GDI SOP revision and review ([7.3](#73-gdi-sop-review-and-revision)) in the GDI SOP GitHub repository.
```
Comment (elisavettorstensson) [inline docs/GDI-SOP_ISM.md:125]: I assume this sentence does not only refer to the text for section 7.1.3, correct? It would be helpful to include it in other sections as well, such as 7.1.2, or in a part with more general information.

**Review by elisavettorstensson — COMMENTED** [review]
I think you did a great job transferring the text from the Google Doc to Markdown! I especially liked the figures—they really help to clarify the text. Well done!

I’ve made a few comments on areas that might need some adjustments. I noticed that not all parts of the SOP were completed in the Google Doc, and they’re still missing here. I’m not sure if we need to finalize sections like 7.3 and 7.4 before we release the SOP. Overall, though, you’ve done excellent work with lots of potential for further improvement! :-)

File: docs/GDI-SOP_ISM.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+1. After a SOP request is received in the repository, a OC/SCDP member evaluates the request. If it is a valid request,the OC/SCDP member prepares the SOP draft from the [GDI General SOP template.](./GDI-SOP_sop-template.md)
+2. OC/SCDP member shares template with the authors( OC/SDPC or nominated experts) to write the SOP (i.e. fill in the content)
+3. After the drafting is completed, the OC/SCDP member shares the draft internally or with experts to review for completeness.
+4. Both committees (OC/SDPC) approve the reviewed document.
+5. The OC/SDPC member then shares the approved SOP with GDI MB for authorization.
+6. GDI MB reviews the SOP and authorise or requests changes (repeat from Step 3)
+7. OC/SDPC accessions ([7.2](#72-gdi-sop-accessioning-and-file-namin)) the authorised SOP according to agreed process and the SOP goes into production
+8. OC/SDPC initiates periodic review cycle (repeat Steps 3-6)
+
+![Valid: Proposal for creating and approving European-level SOPs.](images/GDI-ISM_schema_European_Level.png)
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:87]: Another comment on the picture is that the last two bits, cloning and reviewing, sound to me like the OC/SDPC reviews the "instances" from the nodes, rather than the SOP "instance" that lives in the main repository.

I would probably modify the diagram last bit, since the review occurs in parallel with the cloning, and is not dependent on it.

File: docs/images/.DS_Store
Comment (M-casado) [inline docs/images/.DS_Store:1]: This file should not be included in the PR.

File: docs/images/.gitkeep
Comment (M-casado) [inline docs/images/.gitkeep:1]: This file should not be included in the PR.

File: docs/GDI-SOP_ISM.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+6. GDI MB reviews the SOP and authorise or requests changes (repeat from Step 3)
+7. OC/SDPC accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP template according to agreed process and informs nodes ready to adapt to their own node processes.
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Template.png)
+
+_Figure 2: Proposal for creating and approving node-specific SOP templates._
+
+#### 7.1.3 Node-specific SOPs
+
+For each Node-specific SOP, first a template is created and approved at the European level ([7.1.2](#712-node-specific-sop-templates)). Once approved, the template is then used by each node to create their own SOP instance ([7.1.3](#713-node-specific-sops)) which is approved at the node level. The Steps below describe how nodes will create their node-specific SOPs.
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:111]:
```suggestion
For each Node-specific SOP, first a template is created and approved at the European level ([7.1.2](#712-node-specific-sop-templates)). Once approved, the template is then used by each node to create their own SOP instance ([7.1.3](#713-node-specific-sops)) which is approved at the node level. The Steps below describe how nodes will create their node-specific SOP instances.

```

File: docs/GDI-SOP_ISM.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+Steps:
+
+1. OC/SDPC representative(s) from each node copy the SOP template to their node’s SOP management system
+2. OC/SDPC rep (or nominated experts) uses the template to create the SOP according to their node’s needs and internally review for completeness
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:116]:
```suggestion
2. OC/SDPC representative (or nominated experts) uses the Node-specific SOP template (from [7.1.2](#712-node-specific-sop-templates)) to create their node's instance of that SOP, according to their node’s needs, and internally review for completeness

```

File: docs/GDI-SOP_ISM.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+3. OC/SDPC rep initiates their node-defined approval process for the node-specific SOP
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:117]:
```suggestion
3. OC/SDPC representative initiates their node-defined approval process for the node-specific SOP instance

```

File: docs/GDI-SOP_ISM.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+4. [Node-appointed approver] reviews the SOP and approves or requests changes (repeat from Step 2)
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:118]: Why is the first between ``[]``?

File: docs/GDI-SOP_ISM.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+5. OC/SDPC members from the node accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP according to agreed process and the SOP goes into production
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:119]:
```suggestion
5. OC/SDPC member(s) from the node accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP according to the agreed process and the SOP goes into production at their node.

```

File: docs/GDI-SOP_ISM.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Instance.png)
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:121]: Similar comment as above regarding verb tenses

File: docs/GDI-SOP_ISM.md [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+
+_Figure 2: Proposal for creating and approving node-specific SOP templates._
+
+#### 7.1.3 Node-specific SOPs
+
+For each Node-specific SOP, first a template is created and approved at the European level ([7.1.2](#712-node-specific-sop-templates)). Once approved, the template is then used by each node to create their own SOP instance ([7.1.3](#713-node-specific-sops)) which is approved at the node level. The Steps below describe how nodes will create their node-specific SOPs.
+
+Steps:
+
+1. OC/SDPC representative(s) from each node copy the SOP template to their node’s SOP management system
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:115]: Here I would add a reference to the following document: https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_github-management.md

We should let it be known that their management system can very well be GitHub, so these can be cloned easily.

File: docs/GDI-SOP_ISM.md [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+2. OC/SDPC rep (or nominated experts) uses the template to create the SOP according to their node’s needs and internally review for completeness
+3. OC/SDPC rep initiates their node-defined approval process for the node-specific SOP
+4. [Node-appointed approver] reviews the SOP and approves or requests changes (repeat from Step 2)
+5. OC/SDPC members from the node accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP according to agreed process and the SOP goes into production
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Instance.png)
+
+_Figure 3: Proposal for creating and approving node-specific SOPs instances._
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:123]:
```suggestion
_Figure 3: Proposal for creating and approving node-specific SOP instances._

```

File: docs/GDI-SOP_ISM.md (hunk 1 above)
Comment (M-casado) [reply docs/GDI-SOP_ISM.md:125]: Agreed.

File: docs/GDI-SOP_ISM.md [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+
+GDI-SOP00001.v1_Node-Helpdesk-Ticket-Management-Template-SWE.v1
+
+**Supporting GDI SOP documents** will also be assigned an accession number…
+
+Filenames for GDI SOP supporting documents must start with “GDI-SOP\_”, followed by a brief text describing the document, a full stop, and finally a version number. For example, the first approved version of the GDI SOP Charter could be named “GDI-SOP_Charter.v1”.
+
+### 7.3 GDI SOP review and revision
+
+<_To do- describe a general plan for how GDI SOP templates, node instances of SOPs, and supporting documentation will be reviewed and revised. Include guidance for how to update “Document History” sections of all these documents._>
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:143]: I would have liked for this to be elaborated slightly, we could at least mention the checklist I created here: https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_review-checklist.md

File: docs/GDI-SOP_ISM.md [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+4. [Node-appointed approver] reviews the SOP and approves or requests changes (repeat from Step 2)
+5. OC/SDPC members from the node accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP according to agreed process and the SOP goes into production
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Instance.png)
+
+_Figure 3: Proposal for creating and approving node-specific SOPs instances._
+
+SOP template reviews and revisions should follow guidelines for GDI SOP revision and review ([7.3](#73-gdi-sop-review-and-revision)) in the GDI SOP GitHub repository.
+
+### 7.2 GDI SOP accessioning and file naming
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:127]: I think this section is already covered at https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_sop-accessioning.md

Besides, this text is outdated and should be updated if we were to leave it as it is.

File: docs/GDI-SOP_ISM.md [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+
+**Supporting GDI SOP documents** will also be assigned an accession number…
+
+Filenames for GDI SOP supporting documents must start with “GDI-SOP\_”, followed by a brief text describing the document, a full stop, and finally a version number. For example, the first approved version of the GDI SOP Charter could be named “GDI-SOP_Charter.v1”.
+
+### 7.3 GDI SOP review and revision
+
+<_To do- describe a general plan for how GDI SOP templates, node instances of SOPs, and supporting documentation will be reviewed and revised. Include guidance for how to update “Document History” sections of all these documents._>
+
+### 7.4 GDI SOP access rules
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:145]: Any idea on what this could be representing? Should we elaborate, or remove?

File: docs/GDI-SOP_ISM.md [hunk 14]
```diff
@@ -0,0 +1,176 @@
...
+
+_&lt;to-do>_
+
+## 8 References
```
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:149]: Depending on the review, we shall add more references here

File: docs/images/IMS_fig1.png
Comment (M-casado) [inline docs/images/IMS_fig1.png:1]: If this image is not referenced anywhere, I think it should not be added to the repository. Your other diagrams replace them better.

File: docs/images/IMS_fig2.png
Comment (M-casado) [inline docs/images/IMS_fig2.png:1]: If this image is not referenced anywhere, I think it should not be added to the repository. Your other diagrams replace them better.

File: docs/images/IMS_fig3.png
Comment (M-casado) [inline docs/images/IMS_fig3.png:1]: If this image is not referenced anywhere, I think it should not be added to the repository. Your other diagrams replace them better.

**Review by M-casado — CHANGES_REQUESTED** [review]
Happy to see the ISM document added, and well done especially with the diagrams.
I do have a few requests to the PR though that we can address below.

File: docs/GDI-SOP_ISM.md
Comment (M-casado) [inline docs/GDI-SOP_ISM.md:1]: Also please rename the document to ``GDI-SOP_information-service-management.md`` so that it fits with the repo style and other references

File: docs/GDI-SOP_ISM.md [hunk 15]
```diff
@@ -0,0 +1,176 @@
...
+| **Author**   | Mallory Freeberg      | Task 4.3 member | EMBL-EBI                |
+| **Author**   | Marcos Casado Barbero | Task 4.3 member | EMBL-EBI                |
+| **Author**   | Silvia Bahena         | Task 4.3 member | EMBL-EBI                |
+| **Reviewer** | Jeroen Belien         | GDI-Member      | Health-RI/Amsterdam UMC |
+| **Reviewer** | Mattias Strömberg     | Task 4.3 member | UU                      |
+| **Reviewer** | Elisavet Torstensson  | Task 4.3 member | UU                      |
+| **Approver** |                       |                 |                         |     
+| **Approver** |                       |                 |                         | 
+
+## 4 Purpose
```
[the remark this answers is not in the document]
Comment (GabiRinck) [reply docs/GDI-SOP_ISM.md:54]: I think it would be good to keep the numbering format consistent across the SOPs. The SOP0007 for examples uses 1., 2., 3. etc too and this would also go nicely with your sub-section numbering like 7.1.1 ...

**Review by GabiRinck — COMMENTED** [review]
Very clear, easy to read & nice figures. :-) 
I added some suggestions for the sections that were empty - hope it helps.
Many thanks!!

File: docs/GDI-SOP_ISM.md [hunk 16]
```diff
@@ -0,0 +1,176 @@
...
+   2. [GDI SOP accessioning and file naming](#72-gdi-sop-accessioning-and-file-naming)
+   3. [GDI SOP review and revision](#73-gdi-sop-review-and-revision)
+   4. [GDI SOP access rules](#74-gdi-sop-access-rules)
+8. [References](#references)
+
+## 1. Document History
+
+| Version | Author(s) | Description of Changes             | Date |
+| ------- | --------- | ---------------------------------- | ---- |
+| v0      |           | First version of document released |      |
```
[the remark this answers is not in the document]
Comment (M-casado) [reply docs/GDI-SOP_ISM.md:25]: Albeit true, that piece of the Accessioning file is referring to SOP files. Document history wasn't envisioned to be _that strict_, but there's no reason we cannot make it so. In my case I added ``v0`` and would add ``v0`` here too just to keep track of changes that were not tracked automatically by GitHub but existed.

File: docs/GDI-SOP_ISM.md [hunk 17]
```diff
@@ -0,0 +1,176 @@
...
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Instance.png)
+
+_Figure 3: Proposal for creating and approving node-specific SOPs instances._
+
+SOP template reviews and revisions should follow guidelines for GDI SOP revision and review ([7.3](#73-gdi-sop-review-and-revision)) in the GDI SOP GitHub repository.
+
+### 7.2 GDI SOP accessioning and file naming
+
+**GDI SOP templates** will be assigned accessions upon being approved. GDI SOPs template accessions will take the form “GDI-SOP00000.v0”, where the number before the full stop will auto increment by 1 and the number after the full stop will indicate the version number. For example, version 1 of the first approved GDI SOP template will be given the accession “GDI-SOP00001.v1”. GDI SOP templates must be referenced by this accession.
```
[the remark this answers is not in the document]
Comment (M-casado) [reply docs/GDI-SOP_ISM.md:129]: The digits of the identifier (e.g., ``0001``) are the numerical ID of the SOP. I'm expecting it won't reach 9999, but it's not a placeholder per se, and should be there.

**Comment by M-casado** [comment]
Just remembered -- Kind reminder to add a line to the CHANGELOG about this new file.

File: docs/GDI-SOP_ISM.md [hunk 18]
```diff
@@ -0,0 +1,176 @@
...
+| SDPC         | Security and Data Protection Committee |
+
+| **Term**                     | **Definition**                                                                               |
+| ---------------------------- | -------------------------------------------------------------------------------------------- |
+| European-level SOP instances | Standard Operating Procedures (SOPs) to be followed for GDI processes at the European level. |
+| Node-specific SOP template   | Templates for SOPs that can be tailored to the specific processes of each node.              |
+
+## 3. Roles and Responsibilities
+
+| Role         | Full name             | GDI/node role   | Organisation            |
```
[the remark this answers is not in the document]
Comment (silviabah) [reply docs/GDI-SOP_ISM.md:43]: I have added as reviewers those who have reviewed the document from version 0 and when it is approved, I will add the name of who approved the PR :)

File: docs/GDI-SOP_ISM.md [hunk 19]
```diff
@@ -0,0 +1,176 @@
...
+The following table defines the abbreviations and terms relevant to GDI SOPs.
+
+| Abbreviation | Description                            |
+| ------------ | -------------------------------------- |
+| OC           | Operations Committee                   |
+| SDPC         | Security and Data Protection Committee |
+
+| **Term**                     | **Definition**                                                                               |
+| ---------------------------- | -------------------------------------------------------------------------------------------- |
+| European-level SOP instances | Standard Operating Procedures (SOPs) to be followed for GDI processes at the European level. |
```
[the remark this answers is not in the document]
Comment (GabiRinck) [reply docs/GDI-SOP_ISM.md:38]: My preference would be without the "instance", if possible.

File: docs/GDI-SOP_ISM.md (hunk 12 above)
Comment (silviabah) [reply docs/GDI-SOP_ISM.md:133]: Should I  remove this section from the document or just refer to the document: https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_sop-accessioning.md

**Review by jmenglund — APPROVED** [review]
Looks very good!

File: .DS_Store
Comment (jmenglund) [inline .DS_Store:1]: This file shouldn't be included in the PR.

Comment (M-casado) [inline .DS_Store:1]: This shouldn't be in the PR, it's something that is being added when you do ``git add .``, or ``commit -a``, I imagine. You can avoid it by adding one by one the files you want to commit, or by restoring this file after adding them all in batch.

**Review by M-casado — CHANGES_REQUESTED** [review]
I like the new image a lot, well done. I haven't gone through all previous changes, but I assume they're resolved 👍 

The only three things I noticed and would request changes:
- You added a new ``.DS_Store`` that shouldn't be in the PR
- Before merging this PR, change the name of the main file as mentioned above
- Add your changes to the ``CHANGELOG`` file following its format (i.e. a line like ``information-service.... ... - Document detailing ...")

**Review by M-casado — CHANGES_REQUESTED** [review]
Just added some suggestions in the new section
